src/containers/Header/Search.jsx

```diff
@@ -17,16 +17,29 @@ import {
   HASH_REGEX,
 } from '../shared/utils'
 import './search.scss'
+import { getTransaction } from '../../rippled/lib/rippled'
+import SocketContext from '../shared/SocketContext'
 
-const getIdType = (id) => {
+const isNFTOrTransactions = async (id, rippledContext) => {
+  try {
+    await getTransaction(rippledContext, id)
+    return 'transactions'
+  } catch (e) {
+    return 'token'
```

Comment:
The key for NFTs should really be something like `nft` or `nftoken`, instead of using `token`. It's a different page type.

Reply:
Agreed

Reply:
do you mean we should change the url to something like `/nft/:id`?

Reply:
yeah i agree

Reply:
also i don't like the name of this function with relation to its return type. this name should return a boolean and be named something like `isTransaction`. Given what this does, it should be named something like `determineTypeOfHash`

Reply:
@ledhed2222 soon we will have AMM Instances added to the list of 64 character hexs that we have to determine the type of.